### PR TITLE
feat(auth): add support for client API keys

### PR DESCRIPTION
**Pull request by satish-ravi**
For [ZH issue](https://app.zenhub.com/workspaces/acquisition-squad-sprint-board-6010683afabec1001a090887/issues/valora-inc/in-house-liquidity/585)

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -332,6 +346,8 @@ export default class FiatConnectClient implements FiatConectApiClient {
         `${this.config.baseUrl}/accounts/${params.fiatAccountId}`,
         {
           method: 'DELETE',
+          headers: this._getAuthHeader(),
+          ...this._getAuthHeader(),
```
Comment (cajubelt) [inline src/index.ts:350]: why do you need both here? shouldn't it just be `headers: this._getAuthHeader(),` like the others?

Comment (satish-ravi) [reply src/index.ts:350]: oops, this got added by accident. Updated

**Review by cajubelt — APPROVED** [review]
🥳

**Review by jophish — APPROVED** [review]
:+1:

**Pull request merged**